src/main/java/com/google/sps/server/HandleSchedules.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2010 Google Inc.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.sps.server;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Collection;
+import java.util.Iterator;
+
+import com.google.gson.Gson;
+import com.google.sps.data.*;
+
+/**
+ * Exports the ordered schedules to secondary calendars on the user's Google calendar    
+ */
+@WebServlet("/handleSchedules")
+public class HandleSchedules extends HttpServlet {
+  
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    try {
+      com.google.api.services.calendar.Calendar client = Utils.loadCalendarClient();
+
+      // Get the ordered schedules
+      // Temporary hard-coded schedule, will be retrieved from Rank.java later on 
+      Schedule schedule = TesterSchedule.singleCourseSchedule();
+      Collection<Schedule> schedules = new ArrayList<>();
+      schedules.add(schedule);
+
+      String calendarId = null;
+
+      //for each schedule, create new secondary calendary and put schedule on it
+      Iterator<Schedule> iterator = schedules.iterator();
+      while (iterator.hasNext()) {
+        calendarId = CreateCalendar.createCalendar(client);
+        AddSchedule.addSchedule(iterator.next(), client, calendarId);
```

Comment:
Instead of creating separate classes `CreateCalendar` and `AddSchedule` with static methods, it might make more sense from an object-oriented programming perspective to create a new class that represents the secondary calendar named `ScheduleCalendar` or something similar. You would be able to store variables like the `calendarId` and have methods like `addSchedule` directly within that class.

Reply:
This does make more sense, thank you for the suggestion. I'll make that change!